src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLSPTextViewConnectionListener.cs

```diff
@@ -63,13 +106,60 @@ public void SubjectBuffersConnected(ITextView textView, ConnectionReason reason,
             }
 
             RazorLSPTextViewFilter.CreateAndRegister(vsTextView);
+
+            // Initialize the user's options and start listening for changes.
+            _razorEditorOptions = _editorOptionsFactory.GetOptions(textView);
```

Comment:
To ensure that we're not being projected you'll probably want to validate that the `textView.TextBuffer` is a RazorLSP buffer. Otherwise we'll be overriding the default options for whatever language is attempting to host us